packages-exp/functions-exp/src/api.ts

```diff
@@ -26,6 +26,7 @@ import {
   useFunctionsEmulator as _useFunctionsEmulator,
   httpsCallable as _httpsCallable
 } from './service';
+import { getExpInstance } from '@firebase/util';
```

Comment:
Maybe we should use a more permanent name. What do you think about `getModularInstance` ?

Reply:
Sounds good.